src/main/java/org/broadinstitute/hellbender/tools/spark/CreateHadoopBamSplittingIndex.java

```diff
@@ -32,6 +34,7 @@
 @CommandLineProgramProperties(summary = "Create a HadoopBam splitting index as well as a bam index from a bam file.",
         oneLineSummary = "create a hadoop-bam splitting index" ,
         programGroup = SparkProgramGroup.class)
+@DocumentedFeature(groupName= HelpConstants.DOCS_CAT_DATA)
```

Comment:
Ah, to be frank I'm less concerned about the appearance of the cli help output, as long as everything is listed there that should be. If people want more elaborately organized content they should see the online docs, that's what they're for.